apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -78,10 +78,63 @@
     "True"
 ]
 
+########################
+# DJANGO
+########################
+
 AUTHENTICATION_BACKENDS = ['django.contrib.auth.backends.ModelBackend', 'apps.dashboard.backend.TupServicesBackend']
 TUP_SERVICES_URL = "https://dev.tup-services.tacc.utexas.edu"
 LOGIN_URL = "/dashboard/login"
 
+########################
+# NEWS / BLOG
+########################
+
+from taccsite_cms.settings import INSTALLED_APPS
+
+tacc_app_index = INSTALLED_APPS.index('taccsite_cms')
+INSTALLED_APPS[tacc_app_index:tacc_app_index] = [
```

Comment:
If these apps are installed (through pip) on Core CMS, why do we need to splice them into `INSTALLED_APPS` in TUP? Do they cause breaking changes when enabled in non-TUP portals?

Reply:
I do not remember.

Nope. _This is exactly how News settings are added in Core-CMS-Resources ([ecep](https://github.com/TACC/Core-CMS-Resources/blob/77ca82d/ecep-cms/settings_custom.py#L101), [texascale](https://github.com/TACC/Core-CMS-Resources/blob/4ef3d84/texascale-org/settings_custom.py#L46)) and in Core-CMS example custom settings ([source](https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/settings_custom.example.py#L87))._

The reason had something to do with order. I will dig and mediate. I want to find the answer, because if this is even necessary, it deserves a comment

Reply:
Well, [when I did it, I labeled it as **a `fix`**](https://github.com/TACC/Core-CMS-Resources/pull/127/commits/0b09af6cb0ae6dd630f9ca94bf36ca9e042a5d81), so I know it must be _**functionally** important_. I do not have record of why beyond that I should "install blog before tacc cms".

Reply:
I have added a comment in a98b955.

Reply:
I guess my real question is, why is this being done in tup-UI settings and not the core CMS settings? Portals that don't need these apps can just not use them, and it would mean we don't have to play jenga with the INSTALLED_APPS setting in downstream deployments.

Reply:
The blog **was** in Core CMS settings. [I had wanted it to stay there.](https://github.com/TACC/Core-CMS/pull/341#discussion_r708454353) It was pulled out in [a settings refactor by a previous employee](https://github.com/TACC/Core-CMS/pull/341).

<details><summary>Why I have not moved it back, and a caveat for me doing so...</summary>

My reason for having kept it out of Core CMS is new and different. I do no want to confuse CMS admins. When https://epoc.global was accidentally installed with a blog, it (understandably) freaked out admins.

I am excited to move the blog-support apps back to Core CMS if Blog does not show up in the admin interface, _at least not for any user but a super user_. I do not want to use valuable time before launch to figure that out.

</details>

How about a comment in code and a JIRA ticket (assigned to me is fine) to move it back to Core CMS and a public promise that I will do it?

Reply:
Wait, I build and deploy `tup-cms` off this branch... so I am in no rush to merge, right… as long as I keep this branch up to date with `main`, this PR can hang out in here until I resolve this problem. If all that is true, then I **retract** [my plea](https://github.com/TACC/tup-ui/pull/98#discussion_r1087366994) to "solve later, merge now".